### PR TITLE
Cleaning the station and growing plants - cyberpunk style

### PR DESCRIPTION
**Pull request by Vexylius**
## About The Pull Request
This pull request ports selected parts of the code from https://github.com/Skyrat-SS13/Skyrat-tg/pull/1843 and this https://github.com/Skyrat-SS13/Skyrat-tg/pull/2778 from Skyrat.

image image
_//I've removed that beaker from jani, it's broken (remove, don't improve)_

## Why It's Good For The Game
More things for MDs to put in people - good

## Changelog
:cl:
add: Janitorial toolset implant
add: Botanical toolset implant
/:cl:

### CODE REVIEW

**Comment by yyzsong** [comment]
Cyberjannie 9000 engage.

File: code/modules/surgery/organs/augments_arms.dm [hunk 1]
```diff
@@ -311,3 +311,20 @@
...
+	desc = "A rather simple arm implant containing tools used in gardening and botanical research."
+	items_to_create = list(/obj/item/cultivator, /obj/item/shovel/spade, /obj/item/hatchet, /obj/item/gun/energy/floragun, /obj/item/plant_analyzer, /obj/item/storage/bag/plants, /obj/item/storage/bag/plants/portaseeder)
+
+/obj/item/organ/cyberimp/arm/janitor
+	name = "janitorial tools implant"
+	desc = "A set of janitorial tools on the user's arm."
+	items_to_create = list(/obj/item/lightreplacer/cyborg, /obj/item/holosign_creator/janibarrier, /obj/item/soap/nanotrasen, /obj/item/reagent_containers/spray/cyborg/drying_agent, /obj/item/mop/advanced/cyborg, /obj/item/paint/paint_remover, /obj/item/reagent_containers/spray/cleaner)
+
+/obj/item/organ/cyberimp/arm/janitor/emag_act(mob/user)
+	for(var/datum/weakref/created_item in items_list)
```
Comment (PowerfulBacon) [inline code/modules/surgery/organs/augments_arms.dm]: ?

**Comment by KubeRoot** [comment]
Not sure if botany arm implant should have floral somatoray, especially since it doesn't use radium to make like the somatoray. And where the heck does it fit a plant bag *and* portable seed extractor?

I think arm implants should be worse than the best tools available for the job - surgery and engineering implants have *two* alternatives, experimental and alien tools, which are either faster or offer more functionality at the cost of having to carry them around. I feel like those implants give you the best tools for the job with the convenience of not having to carry them around.

All that said, I love the idea of more arm implants, heck, I think more and more varied implants in general would be nice.

**Comment by Vexylius** [comment]
Changed botnis set a little. Didn't want to remove his seedmaking ability go away entirely, so I've made the seed thing smaller
image
image
It can hold up to 10 plants of the same type and max 3 different ones. Also I've made a custom sprite for that smaller seedmaker

File: code/modules/surgery/organs/augments_arms.dm (hunk 1 above)
Comment (Vexylius) [reply code/modules/surgery/organs/augments_arms.dm]: I don't know what you mean. It works if that's what you're concerned about

Comment (PowerfulBacon) [reply code/modules/surgery/organs/augments_arms.dm]: Its a pointless for loop that does absolutely nothing. There is nothing inside the for loop

Comment (Vexylius) [reply code/modules/surgery/organs/augments_arms.dm]: Oh, right, it doesn't do anything 😅

**Comment by Crossedfall** [comment]
> I think arm implants should be worse than the best tools available for the job 

Inclined to agree with pretty much this entire comment.

**Review by PowerfulBacon — APPROVED** [review]

**Comment by Vexylius** [comment]
>I think arm implants should be worse than the best tools available for the job

> Inclined to agree with pretty much this entire comment.

While I agree on that, I don't think this applies to this PR specifically, at least now. There isn't simply as much variety in botanists and janitors arsenal to put it somewhere, where it's slightly worse than the best tools for the job (botany literally doesn't have any upgrades for its tools and if you count that seed thingy, it was worse than the machine to begin with, but I've nerfed to be even worse. As for the janitor you really wanna take that slightly faster, self-wetting mop from them?).

Arm implants are based on cyborg loadouts and even use same items. This is true for implants that this PR adds (at least for janitors implant, as there is no botanist borg module). Janitor still needs to carry some trash bag to pick up trash, can ride his janicart with that floor buffer upgrade so he doesn't have to mop at all (and if all dirty turfs are empty (that is without any machines or tables), this and cleaner grenades are the best tools for the job) and can also use their backpack cleaner tank. As said earlier, botanists don't have any tool upgrades. Either they have tools or not. Still have to water the plants with some bucket, so it doesn't take away the need for everything.

Current loadouts:
image image

Please just merge it 😩

**Review by Crossedfall — APPROVED** [review]

**Pull request merged**